WTWSMS/history/characters/isaurian.txt

```diff
@@ -1,5 +1,5 @@
 154001 = {
-	name="Longinius"
+	name="Longinus"
```

Comment:
Latin vs Anglicised, no?

Reply:
Latin form is "Longinus" and "Longinus" is used in most(all?) Egnlish sites

Reply:
While both Longinius and Longinus are Latin names, I think in English Longinus is preferred. Yet it doesn't matter here.